kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -0,0 +1,26 @@
+from kentik_api.public.manual_mitigation import ManualMitigation
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass()
+class CreateRequest:
+    ipCidr: str
+    Comment: Optional[str]
+    platformID: str
+    methodID: str
+    minutesBeforeAutoStop: str
+
+
+class CreateResponse:
+    def __init__(self, result: str) -> None:
+        self.response = {"result": result}
```

Comment:
I think you dont need to keep "self.response" as a dict if it always holds only single key-value pair

Reply:
ack